experiments/generic/pod_network_corruption/pod_network_corruption_ansible_logic.yml

```diff
@@ -0,0 +1,73 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars:
+    c_experiment: "pod-network-corruption"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    c_container: "{{ lookup('env','TARGET_CONTAINER') }}"
+    n_corruption: "{{ lookup('env','NETWORK_PACKET_CORRUPTION_PERCENTAGE') }}"
+    n_interface: "{{ lookup('env','NETWORK_INTERFACE') }}"
+    c_lib: "{{ lookup('env','LIB') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_kind: "{{ lookup('env','APP_KIND') }}"
```

Comment:
True, I do not use it ... I copied the experiment logic from the network-loss and network-latency experiments. They do not use this varieble either, do they?

Reply:
Yeah. Not yet :face_with_head_bandage: The idea with this is to be able to perform `kind` specific status checks. For example: 

```yaml
- name: Obtain the number of replicas.
  shell: kubectl get statefulset -n {{ app_ns }} -l {{ app_label }} -o custom-columns=:spec.replicas
  args:
    executable: /bin/bash
  register: rep_count
  until: "rep_count.rc == 0"
  delay: 60
  retries: 15

- name: Obtain the ready replica count and compare with the replica count.
  shell: kubectl get statefulset -n {{ app_ns }} -l {{ app_label }} -o custom-columns=:..readyReplicas
  args:
    executable: /bin/bash
  register: ready_rep
  until: "ready_rep.rc == 0 and ready_rep.stdout|int == rep_count.stdout|int"
  delay: 60
  retries: 15
```

Reply:
The general status check today using the below is not bad either. But, felt above to be a stronger check. 

```yaml
- name: Checking whether application pods are in running state
  shell: kubectl get pods -n {{ a_ns }} -l {{ a_label }} -o custom-columns=:.status.phase --no-headers
  register: result
  until: "((result.stdout.split()|unique)|length) == 1 and 'Running' in result.stdout"
  delay: '{{ delay }}'
  retries: '{{ retries }}'
```

Reply:
Another reason why the APP_KIND is useful, is to add **verification steps to confirm occurrence of chaos itself** via modified `resourceVersion` on the `kind` (though it may not apply to network tests).

Reply:
The values supported by the APP_KIND will be controlled/enforced via the openAPI validation that will come in via this PR: https://github.com/litmuschaos/chaos-operator/pull/104

Reply:
Based on this, would leave it in the experiments. Maybe a comment (to the effect that it is not used presently but has future use) in the logic file will be useful